README.md

```diff
@@ -1,3 +1,10 @@
+<!--
```

Comment:
I don't think we should have these headers. The README might not be read raw, not in a rendered format, like in tools around NuGet packages, or just when looking in the repo.
If someone changes the README only and not the template, we will get a failed test so we're protected. Also there's not that many contributors apart from us and new ones will learn easily.

Reply:
done, removed this